.travis.yml

```diff
@@ -41,7 +41,7 @@ jobs:
     - name: "Run all tests from all other packages"
       env: TEST_SUITE=non_srv_pkg_tests
     - name: "Compile with older Go release"
-      go: "1.19.12"
```

Comment:
If the `go.mod` file has the `go 1.20` directive then I don't think it will build with Go 1.19 anyway. If we consider Go 1.21 to be our "current" toolchain then Go 1.20 is the "old" toolchain.

Reply:
it should still build I think, the older runtime will ignore the version in the go.mod